js/components/Countdown.js

```diff
@@ -0,0 +1,29 @@
+const React = require("react");
+
+class Countdown extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			time: props.start || 10,
+		};
+
+		setInterval(() => {
+			if (this.state.time === 0) {
+				this.setState(() => {
```

Comment:
setState doesn't take a function as an argument. It takes in an object that you want to be the state changes. If you want to call the alert here, just call the alert. Don't call setState.